osidb/tests/test_flaw.py

```diff
@@ -404,29 +422,6 @@ def test_delegated_affects(self):
         )
         assert undelegated_affect.delegated_resolution is None
 
-        # Migrated/duplicate trackers should be ignored
-        affect = AffectFactory(
-            affectedness=Affect.AffectAffectedness.AFFECTED,
-            resolution=Affect.AffectResolution.DELEGATED,
-            ps_module=ps_module.name,
-            flaw__embargoed=False,
-        )
-        TrackerFactory(
-            affects=(affect,),
-            status="closed",
-            resolution="migrated",
-            embargoed=affect.flaw.is_embargoed,
-            type=Tracker.TrackerType.BUGZILLA,
-        )
-        TrackerFactory(
-            affects=(affect,),
-            status="closed",
-            resolution="won't do",
-            embargoed=affect.flaw.is_embargoed,
-            type=Tracker.TrackerType.BUGZILLA,
-        )
-        assert affect.delegated_resolution == Affect.AffectFix.WONTFIX
```

Comment:
We could still leave this test, as it still checks that migrated/duplicate resolutions are ignored when in conjunction with another valid resolution

Reply:
@dmonzonis I am not removing it, I just moved the trackers to be ignored at the very top of the method to make sure that they get ignored all the time till the end. Does it make sense?

Reply:
You're right, my bad, all good then!